package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ludens-reklame/react-crisscross",
-  "version": "1.0.1",
+  "version": "1.1.0",
```

Comment:
The version number should still not be changed manuelly through PRs. The master branch can still have code thats not released yet. The versioning gets done through an actual release using ‘yarn publish’. 

The release process looks like this:
1. Yarn publish
2. yarn prompts for new version
3. Git push —tags && git push